plugins/woocommerce-admin/client/products/hooks/use-product-variation-navigation.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { PartialProduct, ProductVariation } from '@woocommerce/data';
+import { getNewPath, getPersistedQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import { SimpleNavigationProps } from '../shared/simple-navigation';
+
+export default function useProductVariationNavigation( {
+	product,
+	productVariation,
+}: UseProductVariationNavigationInput ): UseProductVariationNavigationOutput {
+	const { variations } = product;
+	const variationIds = variations ?? [];
+
+	const currentIndex = variationIds.indexOf( productVariation.id ?? -1 );
+	const canNavigateBack = currentIndex > 0;
```

Comment:
Nitpicky, but maybe use `Prev` instead of `Back` to mirror `canNavigateNext` and avoid ambiguity.

```suggestion
	const canNavigatePrev = currentIndex > 0;
```

Reply:
done